src/main/java/maksym/fedorenko/bookstore/dto/BookDto.java

```diff
@@ -0,0 +1,21 @@
+package maksym.fedorenko.bookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {
```

Comment:
since we're using java 17 - it is a great idea to use records for DTO :)

Reply:
Can't add record, checkstyle throws an error, when I add record

Reply:
@RedArtelerist weird stuff...